metron-platform/metron-pcap-backend/src/test/java/org/apache/metron/pcap/query/PcapCliTest.java

```diff
@@ -166,12 +166,12 @@ public void runs_fixed_pcap_filter_job_with_full_argument_list_and_default_datef
     PcapOptions.START_TIME_MS.put(config, 500L);
     PcapOptions.END_TIME_MS.put(config, 1000L);
     PcapOptions.NUM_RECORDS_PER_FILE.put(config, 1000);
+    PcapOptions.PRINT_JOB_STATUS.put(config, true);
```

Comment:
Probably remove these as they're not used anymore by the test.

Reply:
This needs to be there since the print status command line argument is set.

Reply:
Ah, I see. Missed that with the diff only showing the option line in the latest commit. Full diff shows the extra option in the cli options arg list.